tex/latex/fks/fksbase.cls

```diff
@@ -133,7 +133,7 @@
 }
 
 \AtBeginDocument{%
-  \renewcommand\figurename{Obr.}
+  \renewcommand\figurename{\lngopt{cs}{Obr.}\lngopt{en}{Fig.}}
```

Comment:
Je-li použit balík `fkslang`, tak je v rámci inicializace nastaven defaultní jazyk na češtinu (není-li nastaveno jinak). Bez `fkslang` není makro `\lngopt` dostupné.